docs/migration.md

```diff
@@ -26,13 +260,52 @@ your existing clients:
 3. Should access tokens granted to this client have only access to the associated user?
   1. Add scope `ME`
 4. Remove the old scopes (`GET`, `POST`, `PUT`, `PATCH`, `DELETE`)
-5. Deploy OSIAM
 
-**NOTE:** The new scopes `ADMIN` and `ME` have been supported since the last
-release, to make it possible to just switch to them without upgrading OSIAM and
-still have everything work as expected. This way you can schedule an upgrade at
-your own convenience and minimize downtime.
+**WARNING:** Adding the scope `ADMIN` to an untrusted client can lead to security issues.
+Access tokens granted to it will have full access to OSIAM.
+
+**NOTE:** Remember to change your applications, too, so they use the new scopes.
+
+The new scopes `ADMIN` and `ME` are supported since version 2.1.
+This way it is possible to switch without updating OSIAM and still have everything work as expected.
+This way you can schedule an update at your own convenience and minimize downtime.
+
+#### Configure the Connector4Java
+
+If you write Java applications that integrate with OSIAM and are using the Connector4Java,
+then you will have to change the configuration of the connector:
+
+```java
+new OsiamConnector.Builder()
+        [...]
+        .setAuthServerEndpoint(AUTH_ENDPOINT_ADDRESS)
+        .setResourceServerEndpoint(RESOURCE_ENDPOINT_ADDRESS)
+        [...]
+        .build();
+```
+
+becomes
+
+```java
+new OsiamConnector.Builder()
+        [...]
+        .withEndpoint(OSIAM_ENDPOINT)
+        [...]
+        .build();
+```
+
+#### API Changes
+
+There have been some minor API changes, namely:
+
+- Reply with 400 BAD REQUEST to invalid filters.
```

Comment:
Thanks for the rewording. I actually just copied the entry from the changelog :)